src/advanced_inv.cpp

```diff
@@ -958,31 +958,20 @@ bool advanced_inventory::move_all_items()
 
     if( spane.get_area() == AIM_INVENTORY || spane.get_area() == AIM_WORN ) {
         if( dpane.get_area() == AIM_INVENTORY ) {
-            popup( _( "You try to put your bags into themselves, but physics won't let you." ) );
+            popup_getkey( _( "You try to put your bags into themselves, but physics won't let you." ) );
             return false;
         } else if( dpane.get_area() == AIM_WORN ) {
             // TODO: implement move_all to worn from inventory.
-            popup( _( "Putting on everything from your inventory would be tricky.  Try equipping one by one." ) );
+            popup_getkey(
+                _( "Putting on everything from your inventory would be tricky.  Try equipping one by one." ) );
             return false;
         }
     }
 
     if( dpane.get_area() == AIM_WORN ) {
         // TODO: implement move_all to worn from everywhere other than inventory.
-        popup( _( "You look at the items, then your clothes, and scratch your head…  Try equipping one by one." ) );
-        return false;
-    }
-
-    // Check first if the destination area still has enough room for moving all.
-    const units::volume &src_volume = spane.in_vehicle() ? sarea.volume_veh : sarea.volume;
-    units::volume dest_volume_free;
-    if( dpane.container ) {
-        dest_volume_free = dpane.container->get_remaining_capacity();
-    } else {
-        dest_volume_free = darea.free_volume( dpane.in_vehicle() );
-    }
-    if( !is_processing() && src_volume > dest_volume_free &&
-        !query_yn( _( "There isn't enough room.  Attempt to move as much as you can?" ) ) ) {
```

Comment:
Why is this removed?

Reply:
Accidental leftover from a non-pushed change, good catch.